src/Equinor.ProCoSys.IPO.Infrastructure/IPOContext.cs

```diff
@@ -104,9 +104,12 @@ public override async Task<int> SaveChangesAsync(CancellationToken cancellationT
             }
         }
             
-        public async Task<IDbContextTransaction> BeginTransaction(CancellationToken cancellationToken = default) 
+        public async Task BeginTransactionAsync(CancellationToken cancellationToken = default) 
             => await base.Database.BeginTransactionAsync(cancellationToken);
 
+        public async Task RollbackTransactionAsync(CancellationToken cancellationToken = default)
+            => await base.Database.RollbackTransactionAsync(cancellationToken);
+
```

Comment:
@janid1967 Do you see any problems in implementing RollbackTransactionAsync like this?